build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -264,7 +268,7 @@ service ContentAddressableStorage {
   // portion present and omit the rest.
   //
   // * `NOT_FOUND`: The requested tree root is not present in the CAS.
-  rpc GetTree(GetTreeRequest) returns (GetTreeResponse) {
+  rpc GetTree(GetTreeRequest) returns (stream GetTreeResponse) {
```

Comment:
It occurs to me that calling this GetTree, even though it was intended as a custom method (https://cloud.google.com/apis/design/custom_methods), is confusing with the style for Get*. We should probably rename it while we are here to avoid this.